flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/service/StandaloneFlinkService.java

```diff
@@ -115,7 +117,7 @@ protected FlinkStandaloneKubeClient createNamespacedKubeClient(
                 executorService);
     }
 
-    private void submitClusterInternal(Configuration conf, Mode mode)
+    private void submitClusterInternal(Configuration conf, Mode mode, Optional<JobSpec> jobSpec)
```

Comment:
We should use a `@Nullable` arg here instead of an Optional, as per Flink [coding standards](https://flink.apache.org/contributing/code-style-and-quality-java.html#java-optional)